### ISSUE TITLE
Question of dataset: Is the bool_flag in the dataset a sign of "done" in standard RL dataset?

### ISSUE THREAD

**Issue by Twilighthui** [Baichenjia/UTDS#3]
Excuse me, I would like to ask if the sixth "bool_flag" taken from a piece of data is the “done” flag in standard RL data.
For example, bool_flag  in the code "obs, action, reward, discount, next_obs, bool_flag = utils.to_torch(batch, self.device)".The complete code is as follows, 

	def update(self, replay_iter, step, total_step):
		metrics = dict()

		batch = next(replay_iter)
		obs, action, reward, discount, next_obs, bool_flag = utils.to_torch(
			batch, self.device)
		bool_flag = bool_flag.cpu().detach().numpy()

		if self.use_tb:
			metrics['batch_reward'] = reward.mean().item()

		# update critic
		metrics.update(
			self.update_critic(obs, action, reward, discount, next_obs, step, total_step, bool_flag))

		# update actor
		if step % self.policy_freq == 0:
			metrics.update(self.update_actor(obs, action))

		# update actor target
		utils.soft_update_params(self.actor, self.actor_target, self.actor_target_tau)

		# update critic target
		for i in range(self.ensemble):
			utils.soft_update_params(self.critic[i], self.critic_target[i], self.critic_target_tau)

		return metrics

However, in pbrl, "bool_flag " seems to be used to regulate the coefficient, which is not mentioned in the paper. I would like to ask whether "bool_flag "  in the dataset a sign of "done" in standard RL dataset?

		for qf_index in np.arange(self.ensemble):
			ucb_ratio_in_flag = bool_flag * self.ucb_ratio_in + (1 - bool_flag) * self.ucb_ratio_in * self.share_ratio_now
			ucb_ratio_in_flag = np.expand_dims(ucb_ratio_in_flag, 1)
			q_target = reward + discount * (target_q_pred[qf_index] - torch.from_numpy(ucb_ratio_in_flag.astype(np.float32)).cuda() * ucb_next)  
			
I would greatly appreciate it if you could answer my question！
